templates/web/panels/panel.tt.html

```diff
@@ -175,6 +175,21 @@ <h4 class="panel_title">
                 <hr class="floatclear">
                 [% END %]
 
+            [% ELSIF element_type == "action" %]
+                <div>
+                [% action_element = element_ref.action_element %]
+
+                [% action_element.html %]
+
+                [% FOREACH action IN action_element.actions %]
+                    <a class="button small"
+                        href="/cgi/product.pl?type=edit&code=[% product.code %][% IF action == 'add_categories' %]#categories[% END %][% IF action == 'add_ingredients_text' %]#ingredients[% END %][% IF action == 'add_nutrition_facts' %]#nutrition[% END %]" >
```

Comment:
shouldnt we introduce a helper function for that ?
eg. `action_url` 

There are some actions missing here, like edit_nutrition

Reply:
> shouldnt we introduce a helper function for that ?

That's a good idea, I will do that.